PurchasesCoreSwift/Identity/IdentityManager.swift

```diff
@@ -123,8 +123,7 @@ extension IdentityManager {
 
     @objc(createAliasForAppUserID:completion:)
     public func createAlias(appUserID alias: String, completion: @escaping (Error?) -> Void) {
-        guard !currentAppUserID.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty,
-              !alias.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty else {
+        guard !alias.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty else {
             completion(ErrorUtils.missingAppUserIDError())
```

Comment:
maybe we want a different error here now, something saying you're trying to set an empty alias?

Reply:
Added new code and updated tests.